From david-b@pacbell.net Wed Jan 17 06:30:46 2007
From: Petr Stetiar <ynezz@true.cz>
Date: Wed, 17 Jan 2007 06:30:39 -0800
Subject: USB: Fix for typo in ohci-ep93xx.c
To: Greg KH <greg@kroah.com>
Cc: Petr Stetiar <ynezz@true.cz>
Message-ID: <200701170630.39951.david-b@pacbell.net>
Content-Disposition: inline

From: Petr Stetiar <ynezz@true.cz>

Attached patch fixes typo in USB driver reported by Chase Douglas on linux-cirrus mailing
list. http://www.freelists.org/archives/linux-cirrus/12-2006/msg00003.html

Signed-off-by: Petr Stetiar <ynezz@true.cz>
Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ohci-ep93xx.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/host/ohci-ep93xx.c
+++ gregkh-2.6/drivers/usb/host/ohci-ep93xx.c
@@ -169,7 +169,7 @@ static int ohci_hcd_ep93xx_drv_remove(st
 static int ohci_hcd_ep93xx_drv_suspend(struct platform_device *pdev, pm_message_t state)
 {
 	struct usb_hcd *hcd = platform_get_drvdata(pdev);
-	struct ochi_hcd *ohci = hcd_to_ohci(hcd);
+	struct ohci_hcd *ohci = hcd_to_ohci(hcd);
 
 	if (time_before(jiffies, ohci->next_statechange))
 		msleep(5);
